src/pyhf/utils.py

```
@@ -117,3 +123,81 @@ def citation(oneline=False):
+
+    os_version = "Cannot be determined"
+    if sys.platform == "linux":
+        try:
+            # platform.freedesktop_os_release added in Python 3.10
+            # Remove when Python 3.9 support dropped
+            from platform import freedesktop_os_release
+        except ImportError:
+            # c.f. https://docs.python.org/3/library/platform.html#platform.freedesktop_os_release
+            from pathlib import Path
+
+            def freedesktop_os_release():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `freedesktop_os_release`

(ANN202)